src/components/safe-apps/AppCard/index.tsx

```diff
@@ -11,25 +11,56 @@ import { SafeAppData } from '@gnosis.pm/safe-react-gateway-sdk'
 import { SAFE_REACT_URL } from '@/config/constants'
 import useChainId from '@/hooks/useChainId'
 import ShareIcon from '@/public/images/share.svg'
+import BookmarkBorderIcon from '@mui/icons-material/BookmarkBorder'
+import BookmarkIcon from '@mui/icons-material/Bookmark'
+//import DeleteIcon from '@/public/images/delete.svg'
 import { AppRoutes } from '@/config/routes'
+import styles from './styles.module.css'
+
+export type SafeAppCardVariants = 'default' | 'compact'
 
 type AppCardProps = {
   safeApp: SafeAppData
+  pinned?: boolean
+  onPin?: (appId: number) => void
+  variant?: SafeAppCardVariants
+}
+
+type CompactSafeAppCardProps = {
+  safeApp: SafeAppData
+  url: string
+  pinned?: boolean
+  onPin?: (appId: number) => void
 }
 
 type AppCardContainerProps = {
   url?: string
   children: ReactNode
+  variant?: SafeAppCardVariants
+}
+
+const enum AppCardVariantHeights {
+  compact = '120px',
+  default = '180px',
 }
 
-export const AppCardContainer = ({ url, children }: AppCardContainerProps): ReactElement => {
+const enum AppCardVariantAspectRatio {
+  compact = '1 / 1',
+  default = 'auto',
+}
+
+const AppCardContainer = ({ url, children, variant }: AppCardContainerProps): ReactElement => {
+  const height = variant === 'compact' ? AppCardVariantHeights.compact : AppCardVariantHeights.default
+  const aspectRatio = variant === 'compact' ? AppCardVariantAspectRatio.compact : AppCardVariantAspectRatio.default
```

Comment:
Nit: I feel like the enum usage is a bit backwards here. I'd sooner extract `"compact"` and `"default"` into one as it's used across multiple files. I know that MUI uses strings though.